--- a/ggml/src/ggml-cpu/ggml-cpu.c
+++ b/ggml/src/ggml-cpu/ggml-cpu.c
@@ -1352,13 +1352,13 @@
     // nb01 >= nb00 - src0 is not transposed
     //   compute by src0 rows
 
+    const bool src1_cont = ggml_is_contiguous(src1);
+
     // TODO: extract to "extra_op"
 #if GGML_USE_LLAMAFILE
     // broadcast factors
     const int64_t r2 = ne12 / ne02;
     const int64_t r3 = ne13 / ne03;
-
-    const bool src1_cont = ggml_is_contiguous(src1);
 
     if (src1_cont) {
         for (int64_t i13 = 0; i13 < ne13; i13++)
